### PR TITLE
Make Rails admin an Vietnamese CMS

### PR DESCRIPTION
**Pull request by nvpmai95**
- Add Vietnamese attributes to model Student, Teacher
- Make Vietnamese the main language
- Add en and vi .yml files for Rails admin

### CODE REVIEW

**Comment by harley** [comment]
nice! did you have to do the vi.yml for rails_admin yourself?

**Comment by nvpmai95** [comment]
of course not. I took from this https://www.localeapp.com/projects/905

File: Gemfile [hunk 1]
```diff
@@ -67,5 +67,5 @@ group :development, :test do
   gem 'pry-stack_explorer'
   gem 'mina', '~> 1.0', '>= 1.0.2'
   gem 'mina-multistage', require: false
-  gem 'rspec-rails', '~> 3.5'
+  gem 'rspec-rails'
```
Comment (harley) [inline Gemfile]: a good practice is to go to `https://rubygems.org/gems/rspec-rails` and copy the gem syntax there.

File: config/initializers/locale.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Where the I18n library should search for translation files
+I18n.load_path += Dir[Rails.root.join('lib', 'locale', '*.{rb,yml}')]
+
+I18n.config.available_locales = [:en, :vi]
```
Comment (harley) [inline config/initializers/locale.rb]: can probably remove the empty lines in this file

File: Gemfile (hunk 1 above)
Comment (nvpmai95) [inline Gemfile]: yeah, i just had a look at http://guides.rubygems.org/patterns/#pessimistic-version-constraint.
We want to restrict to version 3.5

**Pull request merged**